labcodes/labstorybook
# Cards docs

**Link to task**
https://labcodes.atlassian.net/browse/DSYS-196

**How to test**
With the labsystem running, go to the Card tab and check if everything is according to the documentation.
See https://www.figma.com/file/Wx7jj5klQn3YsRA2XcIV3E/%F0%9F%93%9A-Components?node-id=8904%3A70931 for a detailed reference.

**Description**
This PR translates Card's documentation from Figma to Storybook.
Also changes the` .doc-anatomy-block` color to fix accessibility issues on Card's story and on all the other components stories.
